app/views/Project/PillarAnalysis/index.tsx

```diff
@@ -142,42 +232,26 @@ interface PropsFromDispatch {
 const mapDispatchToProps = (dispatch: Dispatch): PropsFromDispatch => ({
     setPillarAnalysisData: params => dispatch(setPillarAnalysisDataAction(params)),
 });
+*/
 
 type FormType = typeof defaultFormValues;
 
 const statementKeySelector = (d: PartialAnalyticalStatementType) => d.clientId;
 
-interface PropsFromState {
-    pillarId: number;
-    projectId: number;
-    analysisId: number;
-
-    activeProject: ProjectDetails;
-    pillarAnalysis: {
-        id: number;
-        versionId: number;
-        data: FormType;
-        pristine: boolean;
-    } | undefined;
-}
-
-interface PageProps {
-}
-type Props = PageProps & PropsFromState & PropsFromDispatch;
-
-function PillarAnalysis(props: Props) {
+function PillarAnalysis() {
+    const {
+        project,
+    } = useContext(ProjectContext);
     const {
-        pillarId,
-        analysisId,
         projectId,
-        activeProject,
-        pillarAnalysis: pillarAnalysisFromProps,
-        setPillarAnalysisData,
-    } = props;
-
-    const initialValue = pillarAnalysisFromProps?.data ?? defaultFormValues;
+        pillarAnalysisId: pillarId,
+        analysisId,
+    } = useParams<{ projectId: string; pillarAnalysisId: string; analysisId: string }>();
+    const alert = useAlert();
+    // const initialValue = pillarAnalysisFromProps?.data ?? defaultFormValues;
+    const initialValue = defaultFormValues;
 
-    const [versionId, setVersionId] = useState(pillarAnalysisFromProps?.versionId);
+    const [versionId, setVersionId] = useState(0);
```

Comment:
Do we need to change this?

Reply:
Discussed on call: make redux mock instead of partial removal